src/Symfony/Component/HttpFoundation/Tests/File/MimeType/MimeTypeTest.php

```diff
@@ -75,7 +62,7 @@ public function testGuessWithNonReadablePath()
             $this->markTestSkipped('Can not verify chmod operations on Windows');
         }
 
-        if ('root' === get_current_user()) {
```

Comment:
get_current_user() returns the name of the **file** owner of the main script. Not really what we want here...